test/osde2e/suite_test.go

```diff
@@ -86,3 +89,36 @@ var _ = AfterSuite(func() {
 	err := testEnv.Stop()
 	Expect(err).ToNot(HaveOccurred())
 })
+
+func setVars(possibleWN, possibleNS string) {
+	os.Setenv("WATCH_NAMESPACE", possibleWN)
+	os.Setenv("SKIP_FLAKES", "true")
```

Comment:
`os.Setenv()` does return an error on failures. Do you think this should be some we can to be checking for. Even if its to log the error so we can debug at a later date

Reply:
Thanks I've updated it locally, going to push changes when I verify it!